Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -120,11 +120,16 @@ - (void)dealloc
     if (WebCoreObjCScheduleDeallocateOnMainRunLoop(WKWebViewConfiguration.class, self))
         return;
 
-    _pageConfiguration->API::PageConfiguration::~PageConfiguration();
+    [self _protectedPageConfiguration]->API::PageConfiguration::~PageConfiguration();
```

Comment:
Dot-syntax would be nicer (applies to all other place in this PR).
```suggestion
    self._protectedPageConfiguration->API::PageConfiguration::~PageConfiguration();
```

Reply:
I thought Safari liked the dot-syntax better and WebKit preferred the brackets but it's possible I remember wrong. I don't write a lot of ObjC. I know the bracket syntax looks much better when using RetainPtr but for `self` I guess I don't see a strong reason to not use the dot-syntax.